plugins/talk-plugin-moderation-actions/client/containers/ApproveCommentAction.js

```diff
@@ -1,28 +1,16 @@
 import React from 'react';
-import {getErrorMessages} from 'plugin-api/beta/client/utils';
 import {withSetCommentStatus} from 'plugin-api/beta/client/hocs';
-import {notify} from 'plugin-api/beta/client/actions/notification';
 import ApproveCommentAction from '../components/ApproveCommentAction';
-import isNil from 'lodash/isNil';
 
 class ApproveCommentActionContainer extends React.Component {
 
-  approveComment = async () => {
+  approveComment = () => {
     const {setCommentStatus, comment} = this.props;
 
-    try {
-      const result = await setCommentStatus({
-        commentId: comment.id,
-        status: 'ACCEPTED'
-      });
-
-      if (!isNil(result.data.setCommentStatus)) {
-        throw result.data.setCommentStatus.errors;
-      }
-      
-    } catch (err) {
-      notify('error', getErrorMessages(err));
```

Comment:
The framework doesn't automatically fire notifications to notice the user about the error, so this line is still needed. Also `notify` is a redux action and need to be connected.